sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -88,6 +88,17 @@
         </dependencies>
       </plugin>
     </plugins>
+
+    <pluginManagement>
```

Comment:
Looks like there is enough testing that this already failed. So that's good news. To fix the issue that showed up, you'll need another layer of `<plugins></plugins>` between `<pluginManagement>` and `<plugin>`.